Source/VolunteerReporting/Read/CaseReportsForListing/ICaseReportsForListing.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Read.CaseReportsForListing
+{
+    public interface ICaseReportsForListing
+    {
+        Task<IEnumerable<CaseReportForListing>> GetAllAsync();
```

Comment:
Would be nice to have a Task<IEnumerable<CaseReportForListing>> GetLimit(int limit) which fetches the limit last or first items. #463